src/main/java/org/fcrepo/auth/webac/WebACAuthorizationDelegate.java

```diff
@@ -84,12 +86,20 @@ public String toString() {
 
     static {
         final Map<String, URI> map = new HashMap<>();
-        map.put("GET", WEBAC_MODE_READ);
-        map.put("POST", WEBAC_MODE_APPEND);
-        map.put("PUT", WEBAC_MODE_WRITE);
-        map.put("DELETE", WEBAC_MODE_WRITE);
-        map.put("PATCH", WEBAC_MODE_WRITE);
-        map.put("OPTIONS", WEBAC_MODE_READ);
+        // WEBAC_MODE_READ Permissions
+        map.put(ModeShapePermissions.READ, WEBAC_MODE_READ);
+        // WEBAC_MODE_WRITE Permissions
+        map.put(ModeShapePermissions.CREATE_WORKSPACE, WEBAC_MODE_WRITE);
+        map.put(ModeShapePermissions.DELETE_WORKSPACE, WEBAC_MODE_WRITE);
+        map.put(ModeShapePermissions.REGISTER_NAMESPACE, WEBAC_MODE_WRITE);
+        map.put(ModeShapePermissions.REGISTER_TYPE, WEBAC_MODE_WRITE);
+        map.put(ModeShapePermissions.REMOVE, WEBAC_MODE_WRITE);
+        map.put(ModeShapePermissions.REMOVE_CHILD_NODES, WEBAC_MODE_WRITE);
+        map.put(ModeShapePermissions.RESTORE, WEBAC_MODE_WRITE);
```

Comment:
I feel like most of these should be restricted to fedoraAdmin and not available to anyone with acl:WRITE access. I'd remove:
* CREATE_WORKSPACE
* DELETE_WORKSPACE
* REGISTER_NAMESPACE
* REGISTER_TYPE
* RESTORE

But that decision requires those with more ModeShape experience than I.